tests/regression_tests/automl/test_type_inference.py

```diff
@@ -0,0 +1,26 @@
+import json
+
+import pytest
+
+from ludwig.automl.automl import create_auto_config
+from tests.regression_tests.automl.utils import get_dataset_golden_types_path, get_dataset_object, REGISTRY
+
+
+@pytest.mark.parametrize("dataset_name", REGISTRY)
+def test_ludwig_auto_config(dataset_name):
```

Comment:
nit: Add "type inference" to the test name, i.e. `test_auto_type_inference_regression`